Source/Core/Core/DSP/Jit/x64/DSPJitRegCache.h

```diff
@@ -170,14 +170,14 @@ class DSPJitRegCache
   void MovToMemory(size_t reg);
   void FlushMemBackedRegs();
 
-  std::array<DynamicReg, 37> m_regs;
-  std::array<X64CachedReg, 16> m_xregs;
+  std::array<DynamicReg, 37> m_regs{};
+  std::array<X64CachedReg, 16> m_xregs{};
 
   DSPEmitter& m_emitter;
-  bool m_is_temporary;
-  bool m_is_merged;
+  bool m_is_temporary = true;
```

Comment:
Depending on the constructor `m_is_temporary` is set to `true` or `false`. Won't setting it here produce side effect?

Reply:
Per [cppreference](https://en.cppreference.com/w/cpp/language/constructor), starting in C++11,

> If a non-static data member has a default member initializer and also appears in a member initializer list, then the member initializer is used and the default member initializer is ignored

So, it's fine, but maybe unnecessary.